apps/typegpu-docs/src/content/examples/simulation/boids-typed-bindgroups.ts

```diff
@@ -2,12 +2,11 @@
 {
   "title": "Boids (next)",
```

Comment:
Small nitpick, since we're making this one the default.

```suggestion
  "title": "Boids",
```

Also, I would do the following changes:
- Rename `boids.ts` to `boids-experimental.ts`
- Rename `boids-typed-bindgroups.ts` to `boids.ts`
- Delete `boids-typedbuffers.ts`